groclient/experimental.py

```diff
@@ -0,0 +1,115 @@
+                        ],
+                        "series_description": {
+                            "source_id": 26,
+                            "item_id": 3457,
+                            "metric_id": 2540047,
+                            "frequency_id": 1,
+                            "region_id": 100023971,
+                            "unit_id": 36
+                        }
+                    },
+                    {
+                        "data_points": [
+                            {
+                                "timestamp_sec": "1640044800",
+                                "value": 32.73432922363281,
+                                "start_timestamp": "1639958400",
+                                "end_timestamp": "1640044800"
+                            }
+                        ],
+                        "series_description": {
+                            "source_id": 26,
+                            "item_id": 3457,
+                            "metric_id": 2540047,
+                            "frequency_id": 1,
+                            "region_id": 100023990,
+                            "unit_id": 36
+                        }
+                    }
+                ],
+                "meta": {
+                    "version": "local",
+                    "copyright": "Copyright (c) Gro Intelligence",
+                    "timestamp": "Mon, 03 Apr 2023 18:32:53 GMT"
+                }
+            }
+        Parameters
+        ----------
+         metric_id : integer
+            How something is measured. e.g. "Export Value" or "Area Harvested"
+        item_ids : integer or list of integers
+            What is being measured. e.g. "Corn" or "Rainfall"
+        region_ids : integer or list of integers
+            Where something is being measured e.g. "United States Corn Belt" or "China"
+        partner_region_ids : integer or list of integers, optional
+            partner_region refers to an interaction between two regions, like trade or
+            transportation. For example, for an Export metric, the "region" would be the exporter
+            and the "partner_region" would be the importer. For most series, this can be excluded
+            or set to 0 ("World") by default.
+        source_id : integer
+        frequency_id : integer
+        unit_id : integer, optional
+        start_date : string, optional
+            All points with end dates equal to or after this date
+        end_date : string, optional
+            All points with start dates equal to or before this date
+        coverage_threshold: float, optional
+            custom threshold on the coverage of geospatial data
+        stream: bool, optional
+
+        Returns
+        -------
+
+        dict of data_series containing data points and its description as shown in above example.
+
+        """
+        return lib.get_data_points_v2_prime(self.access_token, self.api_host, Experimental.ALLOWED_SELECTIONS,
```

Comment:
and I think it's better to not passing a constant to lib.get_data_points_v2_prime, we can check user selections within lib.get_data_points_v2_prime function

Reply:
done